components/dropdown/test/dropdown-openers.visual-diff.js

```diff
@@ -17,9 +17,15 @@ describe('d2l-dropdown-openers', () => {
 
 	after(async() => await browser.close());
 
+	afterEach(async function() {
+		const dropdown = this.currentTest.value;
+		if (dropdown) await reset(page, dropdown);
+	});
+
 	// test for https://github.com/BrightspaceUI/core/issues/1398
 	it('autoclose', async function() {
 		await open(page, '#autoclose');
+		await new Promise(resolve => setTimeout(resolve, 50));
```

Comment:
For some reason this test became super flaky for me if I didn't wait before starting to tab